lineman/app/models/base_endpoints_interface.coffee

```diff
@@ -0,0 +1,22 @@
+angular.module('loomioApp').factory 'BaseEndpointsInterface', (RestfulClient) ->
```

Comment:
Thanks James. I'm not sure I totally get this yet. Would you mind helping me understand?

I get some of it.. it's nice to have some stuff around RestfulClient, but I don't get why it needs to be knitted into the Records interface if it's not doing anything with the rest of the records. Do invitation records get relationship access to other records or something?